netebpfext/net_ebpf_ext.c

```diff
@@ -757,37 +771,73 @@ net_ebpf_extension_uninitialize_wfp_components(void)
 {
     size_t index;
     NTSTATUS status;
+    int max_retries = 10;
```

Comment:
is max_retries intended to be a constant?  If so, use either `const int max_retries` or a `#define` instead of a mutable variable.